Modules/Microsoft365DSC/DSCResources/MSFT_AADFeatureRolloutPolicy/settings.json

```diff
@@ -0,0 +1,25 @@
+{
+    "resourceName": "AADPolicyFeatureRolloutPolicy",
+    "description": "This resource configures an Azure AD Policy Feature Rollout Policy.",
+    "permissions":    {
+    "graph":  {
+                  "delegated":  {
+                                    "read":  [
+
+                                             ],
+                                    "update":  [
+
+                                               ]
+                                },
+                  "application":  {
+                                      "read":  [
+
+                                               ],
+                                      "update":  [
```

Comment:
Required permissions are Directory.Read.All for Read or Directory.ReadWrite.All for Update